RecoParticleFlow/PFProducer/src/PFCandConnector.cc

```diff
@@ -72,74 +72,70 @@ reco::PFCandidateCollection PFCandConnector::connect(PFCandidateCollection& pfCa
 
   // loop on primary
   if (bCorrect_) {
-    if (debug_) {
-      cout << "" << endl;
-      cout << "==================== ------------------------------ ===============" << endl;
-      cout << "====================         Cand Connector         ===============" << endl;
-      cout << "==================== ------------------------------ ===============" << endl;
-      cout << "====================   \tfor " << pfCand.size() << " Candidates\t  =============" << endl;
-      cout << "==================== primary calibrated " << bCalibPrimary_ << "     =============" << endl;
-    }
+    LogDebug("PFCandConnector::connect") << "pfCand.size()=" << pfCand.size() << "bCalibPrimary_=" << bCalibPrimary_;
 
     for (unsigned int ce1 = 0; ce1 < pfCand.size(); ++ce1) {
       if (isPrimaryNucl(pfCand.at(ce1))) {
         if (debug_)
```

Comment:
This "debug_" should also get removed.
Then, if you do so, you can further modify the interface of PFCandConnector, and remove such "debug_" from the class members, as well as from the constructor (PFCandConnector is only accessed by PFAlgo, therefore you don't have to propagate this change anywhere else)

Reply:
good point, thanks. done.